autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -65,7 +65,8 @@ def fit(train_data, label, tuning_data=None, time_limits=None, output_directory=
             holdout_frac=None, num_bagging_folds=0, num_bagging_sets=None, stack_ensemble_levels=0,
             hyperparameters=None, num_trials=None, search_strategy='random', search_options=None,
             nthreads_per_trial=None, ngpus_per_trial=None, dist_ip_addrs=None, visualizer='none',
-            verbosity=2, **kwargs):
+            verbosity=2, epochs=None, grace_period=None, reduction_factor=None, brackets=None,
```

Comment:
ditto previous comment. Please add a new argument scheduler_options, these arguments are too niche to be their own dedicated arguments.